app/assets/javascripts/scp-igv.js

```diff
@@ -139,42 +139,74 @@ function initializeIgv() {
 
   const igvContainer = document.getElementById('igv-container')
 
-  const genes = $('.queried-gene')
-
   const genomeId = bamsToViewInIgv[0].genomeAssembly
 
   let reference
-  let locus
+  let searchOptions
+  let fallbackLocus
   if (genomeId === 'Macaca_fascicularis_5.0') {
-    locus = ['chr1:1-2']
+    fallbackLocus = 'chr1:1-2'
 
     // To consider:
     //  - Update genomes pipeline to make such files automatically reproducible
     const genomeAnnotationObj = bamsToViewInIgv[0].genomeAnnotation
-    const genomePath = `${genomeAnnotationObj.link.split('/').slice(0, -2).join('%2F')}%2F`
+    const genomePath =
+      `${genomeAnnotationObj.link.split('/').slice(0, -2).join('%2F')}%2F`
     const bucket = genomeAnnotationObj.bucket_id
     const gcsBase = 'https://www.googleapis.com/storage/v1/b/'
     const macacaFascicularisBase = `${gcsBase + bucket}/o/${genomePath}`
 
+    const fasta = 'Macaca_fascicularis.Macaca_fascicularis_5.0.dna.toplevel.fa'
```

Comment:
would this section read easier if all the macaca constants were defined in a single object?  e.g. 
```
const macaca_genome_info = {
 genomeId: 'Macaca_fascicularis_5.0', 
fasta: '...',
  url: '...',
 cytoband: '...'
}
```

Reply:
I started this refactor, but found it's basically redundant with the `reference` object a few lines below, without quite the latter's succinctness.